Automattic/wp-calypso
# Subscriber Stats: Support paid subscriber stats for Odyssey Stats

Related: https://github.com/Automattic/red-team/issues/51

## Proposed Changes

Replace membership API call on Jetpack sites to use local API, so that Paid Subscribers could be supported.


## Testing Instructions

* Build Jetpack and Odyssey Stats
  * `jetpack build plugins/jetpack`
  * `STATS_PACKAGE_PATH=~/A8C/jetpack/projects/packages/stats-admin yarn dev `
* Ensure Paid Subscriber highlight card and chart work in Odyssey Stats (please ignore the negative number in my screenshot, not a concern at the moment)

<img width="1266" alt="image" src="https://github.com/Automattic/wp-calypso/assets/1425433/9908619d-6d83-4b34-ad91-3c132ae6ab33">


## Pre-merge Checklist
